README.md

```diff
@@ -32,4 +32,9 @@ addresses. Alert type and severity are specified per function per contract addre
 ### Transaction Failure Count
 
 This agent monitors blockchain transactions that have failed and are associated with a specific
-contract address. Alert type and severity are both configurable.
\ No newline at end of file
+contract address. Alert type and severity are both configurable.
+
+### Contract Variable Monitor
+
+This agent monitors contract variable values for specified contract addresses. Alert type and severity
+are specified per variable per contract address.
```

Comment:
Perhaps a brief mention of the percent thresholds in the description for this agent template.